src/Hooks/useHttp.js

```diff
@@ -0,0 +1,38 @@
+import {useState, useEffect} from 'react';
+import axios from 'axios';
+
+const useHttp = (url) => {
+
+    const http = 'http://localhost:8080';
```

Comment:
This is not an evraroment variable, you can read aboud it here: https://create-react-app.dev/docs/adding-custom-environment-variables/